### PR TITLE
Fixing #1621

### LINKED ISSUE #1621: New Opportunity for household's non primary contact: record is read only
https://powerofus.force.com/_ui/core/chatter/groups/GroupProfilePage?g=0F980000000CjRe&fId=0D58000002AFily&fromEmail=1

### PR DESCRIPTION
**Pull request by njjc**
# Warning
# Info
# Issues
Fixes #1621

### CODE REVIEW

File: src/classes/OPP_OpportunityContactRoles_TDTM.cls [hunk 1]
```diff
@@ -188,10 +188,6 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
                 //check if our contact lookup field is incorrect and update it
                 if (thisOpp.Primary_Contact__c != cr.ContactId) {
                     
-                    //update existing opp for OPP_OpportunityNaming call
```
Comment (davidhabib) [inline src/classes/OPP_OpportunityContactRoles_TDTM.cls]: why did you add this last pull request, and why now must it go?  I was assuming it was needed to keep thisOpp in sync.

Comment (njjc) [inline src/classes/OPP_OpportunityContactRoles_TDTM.cls]: It was from before I rewrote the naming to just use contact roles. It didn't actually work and we have no way of testing the way the "New Contact Donation" button creates contact roles except manually. I made a story to make a selenium test for this button since we can't test it in apex.
